serveradmin/resources/views.py

```diff
@@ -110,21 +108,36 @@ def index(request):
         for server in Query(filters, attribute_ids):
             hosts[server['hostname']] = dict(server)
 
-    page = int(request.GET.get('page', 1))
-    per_page = int(request.GET.get('per_page', 8))
-    hosts_pager = Paginator(list(hosts.values()), per_page)
+    page = abs(int(request.GET.get('page', 1)))
```

Comment:
With the new comment https://github.com/innogames/serveradmin/pull/82#discussion_r402255885 feel free to just handle it in a safe way and no need to make any defaults.